treeherder/perf/tasks.py

```diff
@@ -10,3 +13,61 @@ def generate_alerts(signature_id):
     newrelic.agent.add_custom_parameter("signature_id", str(signature_id))
     signature = PerformanceSignature.objects.get(id=signature_id)
     generate_new_alerts_in_series(signature)
+
+
+@task(name='check-confirming-perf-alerts')
+def check_confirming_perf_alerts_status():
+    # TODO: what's this for? do I need it?
+    # newrelic.agent.add_custom_parameter("signature_id", str(signature_id))
```

Comment:
This just annotates the new relic logs with extra data, I wouldn't worry about adding lines like this unless you're trying to debug a problem.